lib/Raven/Breadcrumbs/ErrorHandler.php

```diff
@@ -21,16 +21,11 @@ public function __construct(\Raven\Client $ravenClient)
 
     public function handleError($code, $message, $file = '', $line = 0, $context = array())
     {
-        $this->ravenClient->breadcrumbs->record(array(
-            'category' => 'error_reporting',
-            'message' => $message,
-            'level' => $this->ravenClient->translateSeverity($code),
-            'data' => array(
-                'code' => $code,
-                'line' => $line,
-                'file' => $file,
-            ),
-        ));
+        $this->ravenClient->leaveBreadcrumb($this->ravenClient->translateSeverity($code), Breadcrumb::TYPE_ERROR, 'error_reporting', $message, [
```

Comment:
@dcramer I committed a change to request you feedback about the leaveBreadcrumb function now accepting an instance of the breadcrumb class instead of a set of arguments. This way users can create and manage with getters/setters the data of the breadcrumb before recording it